### ISSUE TITLE
Next Album doesn't do what I expected

### ISSUE THREAD

**Issue by madprops** [audacious-media-player/audacious#1462]
Audacious 4.4 QT Interface

I have shuffle with shuffle album.

Next Album goes to another album but with the same  track number.

If I'm at the 3rd track of an album, and do Playback -> Next Album I would expect another album starting at t rack 1 but it starts at track 3.

It should start at track 1.

**Comment by madprops** [comment]
Actually it's not exactly at the same track numbers, it seems random. I have shuffle on but with the "shuffle albums" option.

**Comment by madprops** [comment]
Also middle click the Next button should do Next Album etc.

**Comment by radioactiveman** [comment]
Works fine here. "Playback -> Next album" always starts the first title of another album.

Are you sure the tracks are all tagged with correct track numbers?

**Comment by madprops** [comment]
Yes tags are fine. Also I have Repeat. Shufle. And Shuffle by Album.

I just tried it and it went to the 3rd song of a fully tagged album.

I'm thinking maybe it's avoiding songs it already played perhaps.

**Comment by radioactiveman** [comment]
I still cannot reproduce this, therefore closing.
